src/core/tests/frontend/frontend_manager.cpp

```diff
@@ -15,11 +15,6 @@
 
 using namespace ov::frontend;
 
-static std::string mock_fe_path() {
-    static auto lib_name = std::string(FRONTEND_LIB_PREFIX) + "mock1" + std::string(FRONTEND_LIB_SUFFIX);
-    return ov::util::path_join({CommonTestUtils::getExecutableDirectory(), lib_name});
-}
-
 TEST(FrontEndManagerTest, testAvailableFrontEnds) {
```

Comment:
Suggest not putting helper function into TEST code block, let's only applly disabled flag into TEST functions.